plugins/Jobs.cpp

```diff
@@ -863,8 +864,13 @@ void BedrockJobsCommand::process(SQLite& db) {
         const string& nextRun = result[0][1];
         const string& lastRun = result[0][2];
 
-        // Are we rescheduling?
-        const string& newNextRun = request["repeat"].size() ? _constructNextRunDATETIME(nextRun, lastRun, request["repeat"]) : "";
+        // Passed next run takes priority over the one computed via the repeat feature
+        string newNextRun;
```

Comment:
We should do some validation on what newNextRun contains, since a malformed nextRun could cause the query to fail.

Reply:
True, but we don't really validate this in any other command that receives it, for example [here](https://github.com/Expensify/Bedrock/blob/5987b1e312250f79bca46d988cd16ef5ca0f1794/plugins/Jobs.cpp#L1053) or [here](https://github.com/Expensify/Bedrock/blob/5987b1e312250f79bca46d988cd16ef5ca0f1794/plugins/Jobs.cpp#L509)

Reply:
Also, not quite true, the query would not fail, it would leave the job with a weird nextRun value

Reply:
Good old sqlite everything is a string

Given that we aren't validating timestamps anywhere, and that this wouldnt break the query, this is fine